lambda/index.js

```diff
@@ -36,12 +37,13 @@ function findFirstComputer(devices) {
 }
```

Comment:
this method can be a bit more brief, 
```js
return devices.some((device) => device.type === 'Computer' && device.is_active) ? devicess.find((device) => device.type === 'Computer' && device.is_active) : null 
```
if you don't like it, this code works perfectly well.

Reply:
This is two passes through the list of devices 😛 Ultimately, the list is going to be pretty small, so I don't think it really matters. That being said, it's kind of weird that it does the same check twice.

This checks the existence of a device that is an active computer, and if the existence of one is confirmed, then it finds the first one. It had to find the first one in that first pass anyway. At that rate, I'd rather make a simple for-loop and return early when one is found, and null if nothing is found.

Reply:
wait, ```.find``` returns ```undefined``` if it doesn't find it. If you're willing to change your check to an undefined check instead of null you can do,

```js
devices.find((device) => device.type === 'Computer' && device.is_active)
```